test/test.js

```diff
@@ -61,7 +61,7 @@ console.log( 'test: files not found' );
 
 console.log( 'test: disallowEmptyDocumentation' );
 {
-	let errs = [];
+	const errs = [];
```

Comment:
Commafy this with result below?

    const errs = [],
        result = ...

(+ two other occurrences in this file)